Commit ID: 85c767e771a2265d0f1256856f2eb1efc1b46820
Change ID: zvytxmrwksrmsvktpytpvrkzoptvvurl
Author   : Vince Reuter <vince.reuter@tweag.io> (2026-09-10 10:55:49)
Committer: Vince Reuter <vince.reuter@tweag.io> (2026-09-15 12:09:56)

    fix: bump rules_nodejs to back to 6.3.0

    v6.7.5 was chosen in a recent commit, bringing rules_nodejs up to the most recent
    version available at the time. However, the 6.7.x line of rules_nodejs uses a
    call to use_repo_rule() in the root MODULE.bazel file. With use_repo_rule
    introduced in Bazel 7, this breaks compatibility with Bazel 6. While we seek to
    maintain compatibility with Bazel 6, then, we can continue to rely on
    rules_nodejs 6.3.0.

diff --git a/toolchains/nodejs/MODULE.bazel b/toolchains/nodejs/MODULE.bazel
index 99f8002426..1e8330e81a 100644
--- nodejs/MODULE.bazel
+++ nodejs/MODULE.bazel
@@ -8,7 +8,7 @@
     path = "../../core",
 )
 bazel_dep(name = "platforms", version = "0.0.9")
-bazel_dep(name = "rules_nodejs", version = "6.7.5")
+bazel_dep(name = "rules_nodejs", version = "6.3.0")
 bazel_dep(name = "bazel_skylib", version = "1.6.1")
 
 nix_repo = use_extension("@rules_nixpkgs_core//extensions:repository.bzl", "nix_repo")

